QUEST_LV_0200.tsv

```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
+QUEST_LV_0200_20150317_001931	$Soldier Paulter is agonizing over an order that seems unreasonable to him. Get rid of the monsters nearby on behalf of Soldier Paulter.
+QUEST_LV_0200_20150317_001932	$You defeated all the monsters nearby. Talk with Soldier Paulter again.
+QUEST_LV_0200_20150317_001933	$Defeat the nearby monsters
+QUEST_LV_0200_20150317_001934	$Since you passed the order to Soldier Paulter, go back to Commander Julian and talk to him.
+QUEST_LV_0200_20150317_001935	$Talk to Commander O'Rourke
+QUEST_LV_0200_20150317_001936	$Commander Julian wants you to support O'Rourke's unit, which lost strength due to many battles. Talk to Commander O'Rourke at Veltui Crossroads.
+QUEST_LV_0200_20150317_001937	You defeated Infrogalas that suddenly attacked. Talk with Commander O'Rourke.
+QUEST_LV_0200_20150317_001938	$Defeat the attacking Giant Redwood
+QUEST_LV_0200_20150317_001939	$Talk to the Substitute Owl Sculpture
+QUEST_LV_0200_20150317_001940	$There is an Owl Sculpture that seems to have something to say. Talk to the Substitute Owl Sculpture.
+QUEST_LV_0200_20150317_001941	$Defeat Treegools at Byreti Vacant Lot
+QUEST_LV_0200_20150317_001942	$The Substitute Owl Sculpture wants to help out the soldiers. On behalf of the Substitute Owl Sculpture, defeat Treegools at Byreti Vacant Lot.
+QUEST_LV_0200_20150317_001943	$Report to the Substitute Owl Sculpture
+QUEST_LV_0200_20150317_001944	$You've defeated all Treegools you could find. Report to the Substitute Owl Sculpture.
+QUEST_LV_0200_20150317_001945	$Defeat Treegools
+QUEST_LV_0200_20150317_001946	$Talk with O'Rourke
+QUEST_LV_0200_20150317_001947	$Commander O'Rourke is astonished by your skills after seeing you defeating the monster ambush. Talk with Commander O'Rourke.
+QUEST_LV_0200_20150317_001948	$Look for the soldiers at Niurus Highway
+QUEST_LV_0200_20150317_001949	$Commander O'Rourke is saying that his soldiers who were dispatched to an operation have not returned. Go to Niurus Highway and look for the soldiers.
+QUEST_LV_0200_20150317_001950	$Talk to Soldier Carlos
+QUEST_LV_0200_20150317_001951	$You saved Soldier Carlos who was running away from the monsters. Talk yo Soldier Carlos.
+QUEST_LV_0200_20150317_001952	$Defeat the monsters threatening the soldier
+QUEST_LV_0200_20150317_001953	$Talk to the Settled Owl Sculpture
+QUEST_LV_0200_20150317_001954	$There is a hesitant-looking Owl Sculpture. Talk to the Settled Owl Sculpture.
+QUEST_LV_0200_20150317_001955	$Defeat nearby Woodgoblins
+QUEST_LV_0200_20150317_001956	$The Settled Owl Sculpture wants to repay the soldiers who saved it. To help the soldiers, please defeat nearby Ravinelarvae.
+QUEST_LV_0200_20150317_001957	$You defeated all nearby Ravinelarvae. Talk to the Settled Owl Sculpture.
+QUEST_LV_0200_20150317_001958	$Defeat Ravinelarvae
+QUEST_LV_0200_20150317_001959	$Talk to Soldier Daniels
+QUEST_LV_0200_20150317_001960	$Soldier Daniels is trembling from fear. To look for the other soldiers, Talk to Daniels.
```

Comment:
Daniels -> Dainus 
To look for the other soldiers, Talk to Daniels. -> Talk to Soldier Dainus to look for the other soldiers.